app/renderer/lib/appium-client.js

```diff
@@ -1,9 +1,15 @@
 import _ from 'lodash';
 import Bluebird from 'bluebird';
-import {getWebviewStatusAddressBarHeight, parseSource, setHtmlElementAttributes} from './webview-helpers';
-import {SCREENSHOT_INTERACTION_MODE, APP_MODE} from '../components/Inspector/shared';
+import { parseSource, setHtmlElementAttributes } from './webview-helpers';
+import { SCREENSHOT_INTERACTION_MODE, APP_MODE } from '../components/Inspector/shared';
 
-const {TAP, SWIPE, GESTURE} = SCREENSHOT_INTERACTION_MODE;
+const { TAP, SWIPE, GESTURE } = SCREENSHOT_INTERACTION_MODE;
+
+// Selector for the Android webview - includes the correct top and bottom boundaries
+const ANDROID_WEBVIEW_SELECTOR = '//android.webkit.WebView[1]';
+// Selector for the iOS status bar and Safari address bar - not always present
+const IOS_TOP_CONTROLS_SELECTOR = '**/XCUIElementTypeOther[`name CONTAINS "SafariWindow"`]' +
```

Comment:
not sure if this locator would be reliable enough

Reply:
Yeah, it's definitely not great... but the child elements sadly don't have any other way to identify them, at least in portrait mode. In landscape mode, the third `XCUIElementTypeOther` has a `UnifiedBar` label, but that's about it.
I did however just realise that this selector finds multiple results, so I'll at least add an index in the end.